src/indicators/average_true_range.rs

```diff
@@ -69,15 +73,15 @@ impl AverageTrueRange {
     }
 }
 
-impl Next<f64> for AverageTrueRange {
+impl<'a> Next<f64> for AverageTrueRange {
```

Comment:
Please remove unnecessary liftimes here and in the other places.
(they were removed on purpose a while ago: https://github.com/greyblake/ta-rs/pull/26)